ssh/tailssh/tailssh.go

```diff
@@ -181,7 +181,7 @@ func (c *conn) logf(format string, args ...any) {
 	c.srv.logf(format, args...)
 }
 
-// PublicKeyHandler implements ssh.PublicKeyHandler is called by the the
+// PublicKeyHandler implements ssh.PublicKeyHandler is called by the
```

Comment:
My tooling actually doesn't automatically catch this, and when I've tried to relax it to catch them, I've regretted doing so...